src/views/Pools/components/BountyCard.tsx

```diff
@@ -30,21 +29,17 @@ const StyledCard = styled(Card)`
 
 const BountyCard = () => {
   const { t } = useTranslation()
-  const [estimatedDollarBountyReward, setEstimatedDollarBountyReward] = useState(null)
   const {
     estimatedCakeBountyReward,
     totalPendingCakeHarvest,
     fees: { callFee },
   } = useCakeVault()
+  const cakePriceBusd = usePriceCakeBusd()
+  const cakePriceBusdAsNumber = cakePriceBusd.toNumber()
```

Comment:
The return of `usePriceCakeBusd` has a different reference on each render, even if the num value is the same. Calling `.toNumber()` to get the primitive num value makes it suitable for use in a deps array.